packages/foundations-ts-definitions/scripts/fetch-definitions.js

```diff
@@ -77,6 +77,12 @@ const fetchSchema = async apiVersion => {
         'api-version': apiVersion,
       },
     },
+    {
+      endpoint: `${PLATFORM_API_BASE_URL}/organisations/swagger/latest/swagger.json`,
```

Comment:
This whole object should be removed. Or the code will run, it just won't have anywhere to write to!

Reply:
try to fetch again and get an error, but forgot back to it